com.foss.llamas.poker/lib/build.gradle

```
@@ -20,6 +20,9 @@ dependencies {
     // Use JUnit Jupiter for testing.
     testImplementation 'org.junit.jupiter:junit-jupiter:5.8.1'
 
+    // This is used for parameterized tests.
+    testImplementation 'org.junit.jupiter:junit-jupiter-params:5.0.0-M4'
```

```suggestion
    //https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-params/5.8.1
    testImplementation 'org.junit.jupiter:junit-jupiter-params:5.8.1'
```
Let's make this match `junit-jupiter`.